lib/is.js

```diff
@@ -75,14 +75,18 @@ var util = require('./util');
  * Steam tests.
  */
 (function () {
-    var identifier = '[0-9\\-_]{1,100}';
+    var identifier = '[\\w-]{1,100}';
     def({
         fn: 'steam.id',
         re: new RegExp('^' + identifier + '$', 'i')
     });
     def({
+        fn: 'steam.name',
+        re: new RegExp('^' + identifier + '$', 'i')
+    });
+    def({
         fn: 'steam.profile',
-        re: url('steamcommunity\\.com\\/profiles\\/' + identifier)
+        re: url('steamcommunity\\.com\\/profiles\\/|id\\/.?' + identifier)
```

Comment:
The regex I gave wouldn't allow for neither 😕 It would require either profiles or id to be there.